src/features/projects/components/projectDetails/ProjectInfo.tsx

```diff
@@ -279,12 +279,22 @@ function ProjectInfo({ project }: Props): ReactElement {
                 })}
               </div>
             )}
-
-            <div className={styles.infoText}>
-              {project?.metadata?.siteOwnerName}{' '}
-              {project.metadata.acquisitionYear ? 'since ' : <></>}
-              {project?.metadata?.acquisitionYear}
-            </div>
+            {project?.metadata?.siteOwnerName ||
+            project?.metadata?.acquisitionYear ? (
+              <div className={styles.infoText}>
+                {project?.metadata?.siteOwnerName}{' '}
+                {project?.metadata?.siteOwnerName &&
+                project?.metadata?.acquisitionYear ? (
+                  'since '
+                ) : (
+                  <></>
+                )}
+                {project?.metadata?.acquisitionYear ? 'Since ' : <></>}
+                {project?.metadata?.acquisitionYear}
+              </div>
```

Comment:
This logic results in the word "since" getting repeated if there is both a `siteOwnername` and `acquisitionYear`.

For example, I see the text "Fundacion Magnolios since Since 2016" over [here](https://planet-webapp-867uijql3-planetapp.vercel.app/restoring-the-magnolios-reserve)

Reply:
Not sure if this is in this line or at line 345, but there is a space missing between the word "since" and the following year.

<img width="220" alt="Bildschirmfoto 2022-06-19 um 09 55 29" src="https://user-images.githubusercontent.com/1532418/174471450-d1369296-e8b7-4df2-adcb-5d1f84908ee3.png">

Reply:
Seems to be fixed looking at the latest deployment [here](https://planet-webapp-584bo86bh-planetapp.vercel.app/restoring-the-magnolios-reserve).